examples/mobile/android_example/gradle/libs.versions.toml

```diff
@@ -0,0 +1,20 @@
+[versions]
+agp = "8.3.0"
+junit = "4.13.2"
+junitVersion = "1.1.5"
+espressoCore = "3.5.1"
+appcompat = "1.6.1"
+material = "1.10.0"
+gamesActivity = "2.0.0"
+
+[libraries]
+junit = { group = "junit", name = "junit", version.ref = "junit" }
+ext-junit = { group = "androidx.test.ext", name = "junit", version.ref = "junitVersion" }
+espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "espressoCore" }
+appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "appcompat" }
+material = { group = "com.google.android.material", name = "material", version.ref = "material" }
+games-activity = { group = "androidx.games", name = "games-activity", version.ref = "gamesActivity" }
+
+[plugins]
+androidApplication = { id = "com.android.application", version.ref = "agp" }
+
```

Comment:
```suggestion
```

Reply:
I don't know much about what they will do so I only deleted most files🤔
I'll change the target api to 33 later according to the newest [Play Store's minimum API](https://developer.android.com/distribute/best-practices/develop/target-sdk)